### PR TITLE
Orm3

### LINKED ISSUE #240: implement doctrine 3

### PR DESCRIPTION
**Pull request by pinclau**

### CODE REVIEW

File: config/autoload/doctrine.global.php [hunk 1]
```diff
@@ -54,7 +42,14 @@
             // Override `orm_default` only if you know what you're doing
             'orm_default' => [
                 'class'   => MappingDriverChain::class,
-                'drivers' => [],
+                'drivers' => [
+                    'Frontend\\App\\Entity' => 'AppEntities',
```
Comment (alexmerlin) [inline config/autoload/doctrine.global.php:46]: Why do we need (only) this? If it's needed, shouldn't the `Admin` entities also be listed here?
The app seems to work fine without it.

File: config/autoload/doctrine.global.php [hunk 2]
```diff
@@ -54,7 +42,14 @@
...
+                ],
+            ],
+            'AppEntities' => [
```
Comment (alexmerlin) [inline config/autoload/doctrine.global.php:49]: Same question as line 46.

File: src/App/src/Repository/AbstractRepository.php
Comment (alexmerlin) [inline src/App/src/Repository/AbstractRepository.php:1]: The way I see it, the only thing this class is still useful for is that we don't need to repeat the below annotation in all our repositories:
```php
/**
 * @extends EntityRepository<object>
 */
```
Not sure though, if it's that's enough to keep it.

File: README.md
Comment (alexmerlin) [inline README.md:1]: - in order to help our documentation builder tool with highlighting the code snippets, we should use fenced code blocks
- don't forget to modify the version number for the badge on line 6

File: README.md [hunk 3]
```diff
@@ -10,30 +10,18 @@ DotKernel web starter package suitable for admin applications.
 [![GitHub stars](https://img.shields.io/github/stars/dotkernel/admin)](https://github.com/dotkernel/admin/stargazers)
 [![GitHub license](https://img.shields.io/github/license/dotkernel/admin)](https://github.com/dotkernel/admin/blob/3.0/LICENSE.md)
```
Comment (alexmerlin) [inline README.md]:
```suggestion
[![GitHub license](https://img.shields.io/github/license/dotkernel/admin)](https://github.com/dotkernel/admin/blob/5.0/LICENSE.md)
```

**Review by alexmerlin — CHANGES_REQUESTED** [review]

**Comment by alexmerlin** [comment]
Also, don't forget about [this](https://github.com/dotkernel/admin/pull/252/files#diff-62289115a627786183873ac865bc15295daeea5bfb983b28372649e71278cd8eR12) change from my unmerged PR.

File: config/autoload/doctrine.global.php (hunk 1 above)
Comment (pinclau) [reply config/autoload/doctrine.global.php:46]: Those are the paths where the entities are located for the specific module, in this case for App module, for the Admin module the path is configurated in ConfigProvider of the module

**Review by alexmerlin — APPROVED** [review]

**Pull request merged**